src/windows/post_event.c

```diff
@@ -85,39 +92,6 @@ UIOHOOK_API void hook_post_event(uiohook_event * const event) {
 		}
 	}
 
-	if (event->mask & (MASK_BUTTON1 | MASK_BUTTON2 | MASK_BUTTON3 | MASK_BUTTON4 | MASK_BUTTON5)) {
```

Comment:
So this block of code was here to enforce modifier and button masks on post.  Have you tested sending a shift + the key for A?

Reply:
After thinking about it, the modifiers maybe unneeded.  If you are replaying events, it definitely doesn't make sense because you would send the keydown and up for shift events.

Reply:
Shift+A works, however I just noticed shift+arrow key (selecting text) is broken, while ctrl+arrow key works fine (moving cursor word by word).
This was already broken before this modification.

Reply:
Oh, this looks like a feature :  https://github.com/acartero/libuiohook/blob/bc3f3691ec0b1d2be0a9ef30e2427eab02550dab/src/windows/post_event.c#L259